server/plugin/command.go

```diff
@@ -25,6 +26,11 @@ const (
 	featureStars         = "stars"
 )
 
+const (
+	ErrorNoWebhookFound           = "\nNo webhook was found for this repository or organization. To create one, enter the following slash command `/github setup webhook`"
```

Comment:
2/5 that this is not a global message. It's only used in `handleSubscribesAdd` and can be defined there.